netdata-installer.sh

```diff
@@ -823,7 +823,7 @@ install_go() {
 		download "https://github.com/netdata/go.d.plugin/releases/download/$GO_PACKAGE_VERSION/$GO_PACKAGE_BASENAME" "${tmp}/$GO_PACKAGE_BASENAME"
 
 		download "https://github.com/netdata/go.d.plugin/releases/download/$GO_PACKAGE_VERSION/config.tar.gz" "${tmp}/config.tar.gz"
-		grep "${GO_PACKAGE_BASENAME}" "${installer_dir}/packaging/go.d.checksums" > "${tmp}/sha256sums.txt" 2>/dev/null
+		grep "${GO_PACKAGE_BASENAME}" "${installer_dir}/packaging/go.d.checksums\$" > "${tmp}/sha256sums.txt" 2>/dev/null
```

Comment:
"${installer_dir}/packaging/go.d.checksums" is a path to a file on which grep is looking for line with "${GO_PACKAGE_BASENAME}". Why are you modifying a path to a file?

Reply:
You probably wanted this:
```bash
grep "${GO_PACKAGE_BASENAME}\$" "${installer_dir}/packaging/go.d.checksums" > "${tmp}/sha256sums.txt" 2>/dev/null
```

Reply:
Yes, it was idiotic.